### PR TITLE
Add some examples programs

### PR DESCRIPTION
**Pull request by pontus**
Some examples of how to use crypt4gh as a library.

### CODE REVIEW

**Review by blankdots — APPROVED** [review]

File: examples/keycreator/main.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+// package main for keycreator, an example of what a key creator
+// can look like
```
Comment (jbygdell) [inline examples/keycreator/main.go:2]:
```suggestion
// package main for keycreator, an example of what a key creator can look like
```

File: examples/reader/main.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
+// package main for reader, an example of what a crypt4gh file reader
+// can look like
```
Comment (jbygdell) [inline examples/reader/main.go:2]:
```suggestion
// package main for reader, an example of what a crypt4gh file reader can look like
```

File: examples/writer/main.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
+// package main for writer, an example of what a crypt4gh file writer
+// can look like
```
Comment (jbygdell) [inline examples/writer/main.go:2]:
```suggestion
// package main for writer, an example of what a crypt4gh file writer can look like
```

**Review by jbygdell — APPROVED** [review]
Not a fan of "random" line-breaks in code or comments, especially when it harder to read.

**Pull request merged**